test/scala/com/github/johnynek/bazel_deps/CoursierTest.scala

```diff
@@ -38,4 +38,59 @@ dependencies:
       assert(!uvc.asString.contains("bouncycastle"))
     }
   }
+
+  test("test classifier example") {
+    val config =
+      """
+options:
+  buildHeader: [ "load(\"@io_bazel_rules_scala//scala:scala_import.bzl\", \"scala_import\")" ]
+  languages: [ "java", "scala:2.11.8" ]
+  resolverType: "coursier"
+  resolvers:
+    - id: "mavencentral"
+      type: "default"
+      url: https://repo.maven.apache.org/maven2/
+  transitivity: runtime_deps
+  versionConflictPolicy: highest
+dependencies:
+  net.sf.json-lib:
+    json-lib:jar:jdk15:
+      lang: java
+      version: "2.4"
+"""
+
+    val model = Decoders.decodeModel(Yaml, config).right.get
+    val (graph, shas, normalized) = MakeDeps.runResolve(model, null).get
+    assert(graph.nodes.contains(
+      MavenCoordinate(
+        MavenGroup("net.sf.json-lib"), MavenArtifactId("json-lib", "jar", "jdk15"), Version("2.4"))))
+  }
```

Comment:
@johnynek does testing `graph.nodes.contains` here seem sufficient, or is there something more thorough that would be betteR?

Reply:
This seems great. Can we add a test for exclusion of a non-jar resource (which we advertise support for) and then I'll merge.

Reply:
Done!